src/renderer/components/editable-list/editable-list.tsx

```diff
@@ -71,17 +65,15 @@ export class EditableList<T> extends React.Component<Props<T>> {
       <div className="EditableList">
         <div className="el-header">
           <Input
-            theme="round-black"
-            value={this.currentNewItem}
+            theme="round"
```

Comment:
Why different theme?

Reply:
Because when used within a `<Wizard>` we don't want it to be dark, but I do want the round outlined version.